scripts/extract_benchmarks.py

```diff
@@ -0,0 +1,67 @@
+"""
+Extracts benchmarks data from the JSON output of pytest-benchmark
```

Comment:
Why not just keep the entire JSON? The entire JSON is very big. It contains data for every single iteration (can be 1000's per benchmark). We only want/need the summary statistics.